doc/articles/contributing/guidelines/creating-a-new-release.md

```diff
@@ -0,0 +1,13 @@
+# Guidelines for releasing Uno
```

Comment:
🎗 needs to be added to the TOC. 
💭 maybe this document could be called something else?